### PR TITLE
tests: selenium - try to stabilize kdump test

### PR DESCRIPTION
**Pull request by jscotka**

### CODE REVIEW

File: test/selenium/selenium-kdump.py [hunk 1]
```diff
@@ -83,6 +84,7 @@ def testNFS(self):
                         "Remote over NFS",
                         {"kdump-settings-nfs-mount": "localhost:{}".format(self.share)},
                         )
+        self.refresh(self.frame_name)
```
Comment (martinpitt) [inline test/selenium/selenium-kdump.py]: Indeed I often noticed that the page is out of date. It just looks like this reload is covering up a bug? I. e. the kdump page doesn't update itself properly? Or is there some specific reason why this wouldn't work under selenium?

Comment (jscotka) [reply test/selenium/selenium-kdump.py]: @martinpitt I have no idea, I have a fear that there could be some bug maybe underneath connected with js operations and changing ``kdump`` via cockpit UI.

I've added this page reload to enforce that page is up to date, regarding to current state of kdump service, and not contains some messy info.
When I've tested it locally, changing to NFS kdumps sometimes staled  and timeouts, there is trouble that it regenerates initrd, and in case somegthing fails, could cause kdump issue. but t is not raised as an error.

**Comment by martinpitt** [comment]
It seems that reload [did not help](https://209.132.184.41:8493/logs/pull-13125-20191111-134646-087341bf-cockpit-project-cockpit--fedora-30-selenium-edge/log.html#22), it still fails. Perhaps the timeout is just too small, and regenerating the initramfs takes more than a minute on a busy system?

**Comment by jscotka** [comment]
> It seems that reload [did not help](https://209.132.184.41:8493/logs/pull-13125-20191111-134646-087341bf-cockpit-project-cockpit--fedora-30-selenium-edge/log.html#22), it still fails. Perhaps the timeout is just too small, and regenerating the initramfs takes more than a minute on a busy system?

Yep, that could be an issue too. Bad is that sometimes when I've tested on my machine via `` test/selenium/run-tests --hub $HUB_IP -v -b chrome --sit selenium-kdump.py`` sometimes it is very fast, and sometimes it stale and progress icon is still rotating. There will be some corner case maybe with NFS, what causes it. NFS is not tested inside verify testsuite, so maybe it is not anyhow related to selenium test.

**Comment by jscotka** [comment]
@martinpitt tests passed, could you please review this PR.

File: test/selenium/selenium-kdump.py [hunk 2]
```diff
@@ -65,26 +67,38 @@ def testKdumpBasePage(self):
...
         self.select_by_text(self.wait_id("kdump-settings-location",
                                          cond=clickable),
                             selector_item)
         for textbox_key in textbox_dict:
             location = self.wait_id(textbox_key)
             self.send_keys(location, textbox_dict[textbox_key])
-        self.click(self.wait_text("Apply", cond=visible))
+        self.click(self.wait_text("Apply", cond=clickable))
+        # extend waiting for regeneration of initrd image, may take long time
+        self.wait_id("kdump-settings-dialog", cond=visible, reversed_cond=True, overridetry=self.default_try * 3)
```
Comment (martinpitt) [inline test/selenium/selenium-kdump.py]: Heh, I recently added something similar, but it turned out that it wasn't necessary after all. You can just use `cond=invisible`, right?

File: test/selenium/testlib_avocado/seleniumlib.py [hunk 3]
```diff
@@ -429,9 +434,14 @@ def logout(self):
...
-    def check_machine_execute(self, timeout=3):
+    def check_machine_execute(self, timeout=5):
         try:
             self.machine.execute(command="true", direct=True, timeout=timeout)
-        except subprocess.CalledProcessError:
+        except (subprocess.CalledProcessError, RuntimeError):
             return False
         return True
+
+    def refresh(self, page_frame=None):
```
Comment (martinpitt) [inline test/selenium/testlib_avocado/seleniumlib.py]: This isn't being used any more now?

File: test/selenium/run-tests [hunk 4]
```diff
@@ -32,6 +32,8 @@ CURRENT_DIR = os.path.dirname(os.path.realpath(__file__))
 if CURRENT_DIR not in sys.path:
     sys.path.insert(1, CURRENT_DIR)
 
+TEST_TIMEOUT = 600
```
Comment (martinpitt) [inline test/selenium/run-tests]: can you please prefix this commit with "test:", and add a "Closes #13125" to the commit message?

File: test/selenium/selenium-kdump.py (hunk 2 above)
Comment (jscotka) [reply test/selenium/selenium-kdump.py]: Yes, I've seen your code and I saw them useful, but you've remoted it then :-) theoretically yes as opposite of visible, but probably would be more clear, to wait to dialog window disapper with this. or in case of another conditions, eg. clickable, there is no opossite invisible, but potentially, ``unclickable`` what may be useful (eg. button is there, but impossible to click because you are not root)

File: test/selenium/testlib_avocado/seleniumlib.py (hunk 3 above)
Comment (jscotka) [reply test/selenium/testlib_avocado/seleniumlib.py]: yep, not used now, but maybe will be useful for some other tests and it is pitty to remove it :-)

File: test/selenium/run-tests (hunk 4 above)
Comment (jscotka) [reply test/selenium/run-tests]: yep, done. I hope that in proper format

Comment (martinpitt) [reply test/selenium/run-tests]: The Closes goes at the bottom of commit description, not the Subject line; but as the PR is now just one commit, I can fix that during squashing.
